8bitbench/benchmark.js

```diff
@@ -34,14 +34,24 @@ function dumpFrame(vec) {
 
 class Benchmark {
   isInstantiated = false;
+  romBinary;
+
+  async init() {
```

Comment:
IIUC, this is essentially shared code with the `getBinary` function in `WasmEMCCBenchmark` below, just that there it's only used for initializing the `Module["wasmBinary"] = getBinary(...)` and here it's for an arbitrary (non-Wasm) file, that happens to be the emulator ROM, right?

Can we unify this, ideally also with the mechansim for preloading blogs of JavaScript line items (ARES-6/Babylon below)?